test/bigtest/tests/job-profiles/job-profile-details-test.js

```diff
@@ -69,6 +70,19 @@ describe('Job Profile View', () => {
       expect(jobProfileDetails.profileTree.plusSignButton.isPresent).to.be.false;
     });
 
+    describe('when profile button is clicked', () => {
+      let linkProfileType;
+
+      beforeEach(async function () {
+        linkProfileType = jobProfileDetails.hotLink.$root.dataset.testProfileLink;
+        await jobProfileDetails.hotLink.click();
+      });
+
+      it('redirects to appropriate profile details page', () => {
+        expect(location().pathname).contains(linkProfileType);
```

Comment:
I suggest a comparison with string here, it would be more readable
```suggestion
        expect(location().pathname).contains('/data-import/{linkProfileType}/view');
```

Reply:
What about `endsWith`?

Reply:
Agree, it is better to compare with string as in case the `href` change to the incorrect one the test would still pass as the `location().pathname` reflects its value. In turn, comparing to the string representation would make sure that the link is correct.